autogpt_platform/backend/backend/blocks/airtable/_api_test.py

```diff
@@ -0,0 +1,306 @@
+from os import getenv
+from uuid import uuid4
+
+import pytest
+
+from backend.blocks.airtable._api import (
+    TableFieldType,
+    WebhookFilters,
+    WebhookSpecification,
+    create_field,
+    create_record,
+    create_table,
+    create_webhook,
+    delete_multiple_records,
+    delete_record,
+    delete_webhook,
+    get_record,
+    list_records,
+    list_webhook_payloads,
+    update_field,
+    update_multiple_records,
+    update_record,
+    update_table,
+)
+from backend.sdk import APIKeyCredentials, SecretStr
+
+
+@pytest.mark.asyncio
+async def test_create_update_table():
+
+    key = getenv("AIRTABLE_API_KEY")
+    if not key:
+        return pytest.skip("AIRTABLE_API_KEY is not set")
```

Comment:
Not these tests are bypassed if the api key is not present - we don't want to force them to run on the ci.

The purpose is to test the api with the actual endpoint during development. 

All these tests passed